public/Sjofn/homework3.html

```diff
@@ -0,0 +1,100 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Homework 3</title>
+    <style>
+        img {
+            max-height: 400px
+        }
+        table, th, td {
+            border: 1px solid black;
+            border-collapse: collapse;
+        }
+        tr:nth-child(even) {
+            background-color: #f2f2f2;
+        }
+        td {
+            text-align: center;
+        }
+        tr:not(:first-child):hover {
+            background: url('./assets/bg.jpg') repeat;
```

Comment:
This image doesn't quite fit here, so I suggest you to use a more pale variant of it.
I added and committed an example for you into your branch: 'bg_pale.png', - try using it.